www/plugin.d.ts

```diff
@@ -386,15 +386,15 @@ export interface PurchasesStoreProduct {
     /**
```

Comment:
We might want to delete these compiled files here too (see https://github.com/RevenueCat/react-native-purchases/pull/362)

Reply:
Filed [CSDK-369]